adflow/mphys/mphys_adflow.py

```diff
@@ -608,8 +608,9 @@ def solve_linear(self, d_outputs, d_residuals, mode):
         if mode == "fwd":
             d_outputs["adflow_states"] = solver.solveDirectForRHS(d_residuals["adflow_states"])
         elif mode == "rev":
-            # d_residuals['adflow_states'] = solver.solveAdjointForRHS(d_outputs['adflow_states'])
-            solver.adflow.adjointapi.solveadjoint(d_outputs["adflow_states"], d_residuals["adflow_states"], True)
+            phi = np.zeros_like(d_residuals["adflow_states"])
```

Comment:
Is this turning off the adjoint restart? In pyADflow, we have an option for turning off the restart:
https://github.com/mdolab/adflow/blob/4dda7b34954b066f6b3dabd65abad2aeeed9259e/adflow/pyADflow.py#L4060-L4065